### DISCUSSION TITLE
Library future plans and expectations

### DISCUSSION THREAD

**Discussion by pdpino** [hoangnm/react-native-week-view#269] [Ideas]
Hi @hoangnm

What future plans and/or expectations do you have for this library? I think is a great library! and I would like to continue contributing to it.

If you intend to continue actively developing and growing the library, I would suggest completing some chores:

* [x] Add a LICENSE file (#79)
* [x] Add some basic tests
* [x] Label issues (bug, feature request, high priority, etc)
* [ ] Enrich the README, we could:
  * [x] Add installation instructions
  * [x] Describe key features: supports multiple interactions (swipe through pages, event-press, grid-press, drag-and-drop event) works for both android and iOS, styles are customizable, supports multiple locales, etc
  * [ ] Add more short demos (screenshots, GIFs)
  * [x] Add a section with contributors
* [x] Add github actions enforcing linter and tests for every PR
* [x] Add a contributing guides:
  * [x] Bug issue template
  * [x] Feature Request issue template

We can create new issues with these chores along the way, or even use this as a TODO list.

I would be happy to help with all these stuff!

**Comment by hoangnm** [comment]
Hi @pdpino, I really appreciated your contribution, you helped to improve the library from the performance to the new features! I agree that we should add these stuff, I did plan to add some test cases also. Let's improve these chores!

**Comment by pdpino** [comment]
Great!

Could you give me permissions to label and close issues? I could help keeping that organized.
If you prefer not, that's fine too :slightly_smiling_face:

**Comment by hoangnm** [comment]
No problem! I just granted access to you.

**Comment by pdpino** [comment]
@hoangnm what do you think about me publishing experimental / alpha releases of the library more often? E.g.

- When I implement one or more features I release a v0.XX.0-alpha.1 (unstable) version so some users can test early and give feedback
- At the same time I send a PR to master for review

**Reply by hoangnm** [reply]
That sounds good @pdpino.
